testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCIssuerEndpointTest.java

```diff
@@ -0,0 +1,506 @@
+package org.keycloak.testsuite.oid4vc.issuance.signing;
```

Comment:
Could you add header comments the same as other newly created files?

Reply:
Fixed in  [7beca5e](https://github.com/keycloak/keycloak/pull/27931/commits/7beca5e54dc584c6726a73ca82109f0fc3d22124)

Reply:
Thank you.